docs/pluggable-discovery-specification.md

```diff
@@ -0,0 +1,367 @@
+Discovery tools are a special kind of tool used to find supported boards, a platform developer can create its own
```

Comment:
```suggestion
Discovery tools are a special kind of tool used to find supported boards, a platform developer can create their own
```
I don't think "its" is appropriate here ([reference](https://en.wikipedia.org/wiki/Gender_neutrality_in_languages_with_gendered_third-person_pronouns#It_and_one_as_gender-neutral_pronouns)).

My suggested replacement is debatable, but I think it's reasonable. See the answer to [this question](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Pronouns.html#:~:text=the%20singular%20%E2%80%9Cthey%2C%E2%80%9D) for a reasonable take on the subject.